portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,50 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+private List<String> quotes;
+private ArrayList<String> messages;
+
+  @Override
+  public void init() {
+    quotes = new ArrayList<>();
+    quotes.add(
+        "A ship in port is safe, but that is not what ships are for. "
+            + "Sail out to sea and do new things. - Grace Hopper");
+    quotes.add("They told me computers could only do arithmetic. - Grace Hopper");
+    quotes.add("A ship in port is safe, but that's not what ships are built for. - Grace Hopper");
+    quotes.add("It is much easier to apologise than it is to get permission. - Grace Hopper");
+    quotes.add("If you can't give me poetry, can't you give me poetical science? - Ada Lovelace");
+    quotes.add("I am in a charming state of confusion. - Ada Lovelace");
+    quotes.add(
+        "The Analytical Engine weaves algebraic patterns, "
+            + "just as the Jacquard loom weaves flowers and leaves. - Ada Lovelace");
+    quotes.add(
+        "Sometimes it is the people no one can imagine anything of "
+            + "who do the things no one can imagine. - Alan Turing");
+    quotes.add("Those who can imagine anything, can create the impossible. - Alan Turing");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Shengjing Zhang!</h1>");
+
+    String quote = quotes.get((int) (Math.random() * quotes.size()));
+    response.getWriter().println(quote);
+
+    // Convert the Arraylist to JSON
+    messages = new ArrayList<>();
+    messages.add("Good morning!");
+    messages.add("Good afternoon!");
+    messages.add("Good night!");
+
+    //ServerStats serverStats = new ServerStats(messages.get(0), messages.get(1), messages.get(2));
+    Gson gson = new Gson();
```

Comment:
Please initialize this outside the method as we don't want to initialize this  everytime the method is called. Consider doing in init or directly at global variable declaration. Because we don't need to initialize Gson everytime.